source/components/organisms/SummaryList/SummaryList.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import GroupedList from '../../molecules/GroupedList/GroupedList';
+import SummaryListItem from './SummaryListItem';
+
+export interface Item {
+  title: string;
+  id: string;
+  type: 'number' | 'text' | 'date' | 'arrayNumber' | 'arrayText' | 'arrayDate';
+  category?: string;
+  inputId?: string;
+}
+
+interface Category {
+  category: string;
+  description: string;
+}
+
+interface Props {
+  heading: string;
+  items: Item[];
+  categories?: Category[];
+  onChange: (answers: Record<string, any> | string | number, fieldId: string) => void;
+  color: string;
+  answers: Record<string, any>;
+}
+/**
+ * Summary list, that is linked and summarizes values from other input components.
+ * The things to summarize is specified in the items prop.
+ * The things are grouped into categories, as specified by the categories props.
+ */
+const SummaryList: React.FC<Props> = ({ heading, items, categories, onChange, color, answers }) => {
+  const changeFromInput = (item: Item, index?: number) => (text: string) => {
```

Comment:
Hard to tell what this does from a quick look, add comment or more specific naming

Reply:
Added some more descriptive function comments.